web/sites/default/services.yml

```diff
@@ -0,0 +1,239 @@
+parameters:
```

Comment:
Is there clarity on what's changed? When I adopt a config file in source, I often like to do one commit of the default + additional commit(s) for needs-specific changes